File: conteudo/aulas/tipos/derivados.pas
Program Derivados;
type
   Celcius    = real;
   Fahrenheit = real;
var
   c : Celcius;
   f : Fahrenheit;
begin
   c := 38.0;
   f := 100.0;
   c := f;
   writeln(c);
end.

File: conteudo/aulas/tipos/derivados.c
#include <stdio.h>

typedef float Celcius;
typedef float Fahrenheit;

int main (int argc, char **argv)
{
  Celcius c = 38.0;
  Fahrenheit f = 100.0;
  c = f;
  f = c;
  return 0;
}
